### PR TITLE
Use grey-vest library

### PR DESCRIPTION
**Pull request by michsa**
Draft pending what to do with the versioning on this (breaking change? patch?)

This PR removes the `greyVest` folder and updates the rest of contexture-react to import the components from the [new grey-vest library](https://github.com/smartprocure/grey-vest).

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Your PR has lint errors. Please fix these and commit them.
      
<details>
  <summary>TagsQuerySearchBar.js</summary>
  <p><a href="https://github.com/smartprocure/contexture-react/blob/feature/grey-vest/src/exampleTypes/TagsQuerySearchBar.js" target="_blank">src/exampleTypes/TagsQuerySearchBar.js</a></p>
  
  <strong>'grey-vest' import is duplicated.</strong>
  <code>Line 9: import { TagsInput as ExpandableTagsInput, Tags } from 'grey-vest'</code>
</details>
<details>
  <summary>themePicker.js</summary>
  <p><a href="https://github.com/smartprocure/contexture-react/blob/feature/grey-vest/src/stories/themePicker.js" target="_blank">src/stories/themePicker.js</a></p>
  
  <strong>'grey-vest' import is duplicated.</strong>
  <code>Line 9: import { Flex } from 'grey-vest'</code>
</details>
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>The changelog has not been updated. Please update the changelog.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>The version was not updated. Please update the version.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>:exclamation: This PR is BIG (+359 -2805)  
Please keep it below 500 net changes</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Could not run `duti:fix` command successfully
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>You reduced the total lines of code! Awesome! :+1:</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 41ac07953c0a70f3c79d7947152e9a7a5acc9816
</p>

**Comment by daedalus28** [comment]
So I think we can make this a fix revision and not a major bump. In theory, when we started publishing grey-vest as its own library, we should have made contexture-react take a dependency on it in its package.json but instead effectively did a weird git-based inclusion. Since there won't be any actual API changes, you will be able to safely bump contexture-react in a user project without anything breaking.

The only thing that could be construed as a breaking change is that you can no longer import grey-vest components from contexture-react, which was undocumented and not officially supported (and therefore wasn't part of the public API)

**Comment by daedalus28** [comment]
> So I think we can make this a fix revision and not a major bump. In theory, when we started publishing grey-vest as its own library, we should have made contexture-react take a dependency on it in its package.json but instead effectively did a weird git-based inclusion. Since there won't be any actual API changes, you will be able to safely bump contexture-react in a user project without anything breaking.
> 
> The only thing that could be construed as a breaking change is that you can no longer import grey-vest components from contexture-react, which was undocumented and not officially supported (and therefore wasn't part of the public API)

After discussing this further, it seems like it _should_ actually be a major version bump because we never documented that you can't import grey-vest from contexture-react anymore when we released v2 (and there are probably still stray imports in our primary product anyway).

**Comment by michsa** [comment]
Small problem: GV now has a Text component, and contexture-react already has a Text example type (which I forgot about), so now there are export conflicts.

The options I see:
1. stop exporting GV components from contexture-react now, in this PR (breaking ofc)
2. kill the Text example type since it's weird and we don't need it anyway (easy but obviously breaking; we might as well just stop exporting the components instead)
3. reexport everything but Text from GV
4. rename Text in GV to something else for now (breaking for GV? do we care?)

Option 3 seems like a total pain (unless anyone knows of a magic hack to omit one thing from an `export *`), but is the only solution that doesn't break stuff, so I'll go with it for now and we can discuss alternatives later 🤷‍♀

File: package.json [hunk 1]
```diff
@@ -113,17 +110,14 @@
...
-    "read-pkg": "^5.2.0",
-    "storybook-addon-jsx": "^7.1.0",
-    "write-pkg": "^4.0.0"
+    "react-test-renderer": "^16.2.0"
   },
   "dependencies": {
     "futil": "^1.64.3",
+    "grey-vest": "^3.0.0-alpha.11",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
```
Comment (daedalus28) [inline package.json]: this can die

File: src/themes/greyVest/Style.js [hunk 2]
```diff
@@ -146,6 +146,27 @@ export default () => (
       }
 
 
+      /* CheckButton */
```
Comment (daedalus28) [inline src/themes/greyVest/Style.js]: this can just die

**Pull request merged**